mne/beamformer/_lcmv.py

```diff
@@ -285,6 +353,12 @@ def lcmv(evoked, forward, noise_cov, data_cov, reg=0.05, label=None,
         detected automatically. If int, the rank is specified for the MEG
         channels. A dictionary with entries 'eeg' and/or 'meg' can be used
         to specify the rank for each modality.
+    apply_noise_norm: False | True
+        If True, a noise-normalized beamformer will be computed.
+    weight_norm: None | 'nai' | 'unit_noise_gain*
+        If 'nai', the Neural Activity Index (Van Veen, 1997) will be computed,
+        if 'unit_noise_gain', the unit-noise gain minimum variance beamformer
+        will be computed (Borgiotti-Kaplan beamformer)
```

Comment:
Use `[1]_` and `[2]_` instead of e.g. `Van Veen, 1997` here (but Borgiotti-Kaplan is okay if it's colloquially referred to that way) and `References` section below with `.. [1]` etc. (see other References examples)